config/samples/v2/custom-configs/storage-creds.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: mlpipeline-minio-artifact
+  name: teststoragesecret
```

Comment:
now that we are free from the hardcoded secret name, we should probably include the DSPA's name as a suffix to this Secrets name (future-proofs in the case were multiple DSPAs in the same namespace is supported)